simularium_models_util/actin/actin_util.py

```diff
@@ -1592,6 +1601,43 @@ def add_bonds_between_actins(force_constant, system, util):
             force_constant,
             bond_length,
             system,
+            actin_number_types,
+        )
+        util.add_polymer_bond_1D(
```

Comment:
This bond is the main thing we want to be able to toggle on and off for the longitudinal bonds. Can you add a parameter `longitudinal_bonds` and check that here before adding this new bond?